Runtime/Game/Requests/LootLockerSessionRequest.cs

```diff
@@ -232,20 +251,39 @@ public static void XboxOneSession(LootLockerXboxOneSessionRequest data, Action<L
             }, false);
         }
 
-        public static void AppleSession(LootLockerAppleSignInSessionRequest data, Action<LootLockerSessionResponse> onComplete)
+        public static void AppleSession(LootLockerAppleSignInSessionRequest data, Action<LootLockerAppleSessionResponse> onComplete)
```

Comment:
Is this for starting an apple-session?
I would rename it to StartAppleSession in that case, just "AppleSession" is very vague.

Reply:
Ah, no, this is in another class, thought it was SDKManager, then the naming is fine. Ignore me!